trending_data/src/main/webapp/script.js

```diff
@@ -0,0 +1,14 @@
+let map;
```

Comment:
Very minor nit: I'd move this declaration inside `initMap` so that the linter doesn't complain about unused variables.

Reply:
If this is declared inside init and is not left in the global context, it will produce an [InvalidValueError](https://stackoverflow.com/questions/33641663/why-isnt-this-an-instance-of-map-of-google-maps-map-invalidvalueerror-setmap).